<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>590</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Palatino Linotype</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_Audio">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Grabar Audio </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_RutaAudio">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Grabe Audio Primero</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Evaluar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Evaluar</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>521</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_Audio">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_ReproducirAudio">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Reproducir Audio</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>521</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_Frecuencia">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
   </layout>
  </widget>
  <widget class="QLabel" name="label_Status">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>510</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Resultado</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Hombre">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>350</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Hombre</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Mujer">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>400</y>
     <width>95</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mujer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
